daprdocs/content/en/reference/components-reference/supported-state-stores/setup-mongodb.md

```diff
@@ -47,32 +47,35 @@ spec:
 
 ```yaml
   - name: actorStateStore
     value: "true"
 ```
 
-
 ## Spec metadata fields
 
 | Field              | Required | Details | Example |
 |--------------------|:--------:|---------|---------|
-| server             | Y<sup>*</sup> | The server to connect to, when using DNS SRV record | `"server.example.com"`
-| host               | Y<sup>*</sup> | The host to connect to | `"mongo-mongodb.default.svc.cluster.local:27017"`
+| server             | Y<sup>1</sup> | The server to connect to, when using DNS SRV record | `"server.example.com"`
+| host               | Y<sup>1</sup> | The host to connect to | `"mongo-mongodb.default.svc.cluster.local:27017"`
 | username           | N        | The username of the user to connect with (applicable in conjunction with `host`) | `"admin"`
 | password           | N        | The password of the user (applicable in conjunction with `host`) | `"password"`
 | databaseName       | N        | The name of the database to use. Defaults to `"daprStore"` | `"daprStore"`
 | collectionName     | N        | The name of the collection to use. Defaults to `"daprCollection"` | `"daprCollection"`
 | writeConcern       | N        | The write concern to use | `"majority"`
 | readConcern        | N        | The read concern to use  | `"majority"`, `"local"`,`"available"`, `"linearizable"`, `"snapshot"`
 | operationTimeout   | N        | The timeout for the operation. Defaults to `"5s"` | `"5s"`
-| params             | N<sup>**</sup> | Additional parameters to use | `"?authSource=daprStore&ssl=true"`
+| params             | N<sup>2</sup> | Additional parameters to use | `"?authSource=daprStore&ssl=true"`
 
-> <sup>[*]</sup> The `server` and `host` fields are mutually exclusive. If neither or both are set, Dapr will return an error.
+> <sup>[1]</sup> The `server` and `host` fields are mutually exclusive. If neither or both are set, Dapr will return an error.
 
-> <sup>[**]</sup> The `params` field accepts a query string that specifies connection specific options as `<name>=<value>` pairs, separated by `"&"` and prefixed with `"?"`. e.g. to use "daprStore" db as authentication database and enabling SSL/TLS in connection, specify params as `"?authSource=daprStore&ssl=true"`. See [the mongodb manual](https://docs.mongodb.com/manual/reference/connection-string/#std-label-connections-connection-options) for the list of available options and their use cases.
+> <sup>[2]</sup> The `params` field accepts a query string that specifies connection specific options as `<name>=<value>` pairs, separated by `&` and prefixed with `?`. e.g. to use "daprStore" db as authentication database and enabling SSL/TLS in connection, specify params as `?authSource=daprStore&ssl=true`. See [the mongodb manual](https://docs.mongodb.com/manual/reference/connection-string/#std-label-connections-connection-options) for the list of available options and their use cases.
 
 ## Setup MongoDB
```

Comment:
It would be great if we could point to a resource on how to set up mongoDB replica sets easily as pointed out here https://stackoverflow.com/questions/51461952/mongodb-v4-0-transaction-mongoerror-transaction-numbers-are-only-allowed-on-a. Do you have a suggestion, or better still is there a public container image that could be downloaded and used?

Reply:
Done.

- Docker: I added a link to how to set up a RS using Docker. It's not a simple process, and requires a bunch of steps, so I think it's best to link to the official docs
- K8s: Looks like the Helm chart we recommended for K8s was archived. I've changed the recommendation to use the Helm chart by Bitnami, which also supports configuring Replica Sets. Since there are many options, and it depends on the architecture you choose (single node or RS), I've just added a link